sources/identify.php

```diff
@@ -2115,7 +2115,7 @@ public function getUserInfo($login, $enable_ad_user_auto_creation, $oauth2_enabl
         $oauth2LoginOngoing = isset($session->get('userOauth2Info')['oauth2LoginOngoing']) ? $session->get('userOauth2Info')['oauth2LoginOngoing'] : false;
         $data['oauth2_login_ongoing'] = $oauth2LoginOngoing;
         $data['ldap_user_to_be_created'] = $enable_ad_user_auto_creation === true && DB::count() === 0 && $oauth2LoginOngoing !== true ? true : false;
-        $data['oauth2_user_to_be_created'] = $oauth2_enabled === true && DB::count() === 0 && $oauth2LoginOngoing === true ? true : false;
+        $data['oauth2_user_to_be_created'] = (bool)$oauth2_enabled === true && DB::count() === 0 && $oauth2LoginOngoing === true ? true : false;
```

Comment:
For consistency with the rest of the code, can you add space after closing parenthesis of a cast please?

Reply:
@jhumphries 
Thank you for your PR and fix.
Can you please do the small typo change ask by Corentin in order for me to merge it?